linkis-engineconn-plugins/engineconn-plugins/sqoop/src/main/scala/org/apache/linkis/engineconnplugin/sqoop/launch/SqoopEngineConnLaunchBuilder.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 WeBank
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.linkis.engineconnplugin.sqoop.launch
+
+
+import org.apache.linkis.manager.engineplugin.common.conf.EnvConfiguration
+import org.apache.linkis.manager.engineplugin.common.launch.entity.{EngineConnBuildRequest, RicherEngineConnBuildRequest}
+import org.apache.linkis.manager.engineplugin.common.launch.process.Environment.{HADOOP_CONF_DIR, HIVE_CONF_DIR, PWD, variable}
+import org.apache.linkis.manager.engineplugin.common.launch.process.LaunchConstants.{ENGINE_CONN_CONF_DIR_NAME, ENGINE_CONN_LIB_DIR_NAME, addPathToClassPath}
+import org.apache.linkis.manager.engineplugin.common.launch.process.JavaProcessEngineConnLaunchBuilder
+
+import java.io.File
+import java.util
+import org.apache.commons.lang.StringUtils
+import org.apache.linkis.manager.common.protocol.bml.BmlResource
+import org.apache.linkis.manager.engineplugin.common.conf.EnvConfiguration.{HADOOP_LIB_CLASSPATH, HBASE_LIB_CLASSPATH, LINKIS_PUBLIC_MODULE_PATH}
+
+import java.nio.file.Paths
+import scala.collection.JavaConversions._
+
+class SqoopEngineConnLaunchBuilder extends JavaProcessEngineConnLaunchBuilder{
+  override protected def getBmlResources(implicit engineConnBuildRequest: EngineConnBuildRequest): util.List[BmlResource] = {
+    val filterList: List[String] = List( "lib2.zip",
```

Comment:
lib2.zip? Is this needed, I don't understand the difference between lib2.zip and lib.zip

Reply:
ok,i will delete these and test again.